internal/apps/dop/providers/issue/core/relation.go

```diff
@@ -230,3 +225,18 @@ func (i *IssueService) ValidIssueRelationTypes(ids []uint64, issueTypes []string
 	}
 	return nil
 }
+
+func (i *IssueService) validateAddIssueRelation(req *pb.AddIssueRelationRequest) error {
+	if req == nil {
+		return fmt.Errorf("empty relation request")
+	}
+	if req.Type == IssueRelationInclusion {
+		if err := i.ValidIssueRelationType(req.IssueID, pb.IssueTypeEnum_REQUIREMENT.String()); err != nil {
+			return err
+		}
+		if err := i.ValidIssueRelationTypes(req.RelatedIssues, []string{pb.IssueTypeEnum_REQUIREMENT.String()}); err != nil {
```

Comment:
relatedIssues should not be a `REQUIREMENT`?

Reply:
yes